libs/core/executors/include/hpx/executors/datapar/execution_policy.hpp

```diff
@@ -137,6 +138,9 @@ namespace hpx { namespace execution { inline namespace v1 {
                     HPX_FORWARD(Parameters, params)...));
         }
 
+        using base_policy_type = sequenced_task_policy;
+        static constexpr base_policy_type base_policy = sequenced_task_policy{};
```

Comment:
I'd suggest not to include a pre-constructed instance of the base policy. For one, it's trivial to construct one if needed, but more importantly (and mostly for the _shim policies), constructing a new instance requires access to the concrete executor and parameters objects of the source policy. Those however are not necessarily known at compile time.